designsafe/apps/accounts/views.py

```diff
@@ -331,18 +331,27 @@ def profile_edit(request):
 
     if request.method == 'POST':
         form = forms.UserProfileForm(request.POST, initial=tas_user)
+
         if form.is_valid() and pro_form.is_valid():
+            tas_email_exists = tas.get_user(email=form.cleaned_data['email']) and (tas_user['email'] != form.cleaned_data['email'])
```

Comment:
Please update the unit tests to check for this case (right now `tas_email_exists` always returns `True` in the test context because mocks are truthy). Feel free to ping me if you want to pair up on this!

Reply:
@jarosenb Fixed!